### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by nicolas-raoul** [commons-app/apps-android-commons#4898]
### Summary

When reviewing, at some point the "Yes" and "No" buttons got both disabled and never become enabled in the previous step. 

Skipping the image and trying review another image did not solve the problem, both buttons were still disabled at that step.

![adb](https://user-images.githubusercontent.com/99590/158780276-093219b6-3781-4132-9fe7-547f399b8126.png)

### Steps to reproduce

Difficult to reproduce

### Expected behaviour

/

### Actual behaviour

/

### Android version

8

### Commons app version

master

### Would you like to work on the issue?

Prefer not

**Comment by Aman16-ai** [comment]
can i work on this issue?

**Comment by nicolas-raoul** [comment]
@Aman16-ai Better work on one issue at a time 🙂

**Comment by Eric-peng111** [comment]
@nicolas-raoul Can I work on this issue? thanks

**Comment by nicolas-raoul** [comment]
@Eric-peng111 Did you manage to reproduce the issue? 🙂

**Comment by Eric-peng111** [comment]
No, I did not. Just forked it & cloned to my local, gonna be working on it.

**Comment by nicolas-raoul** [comment]
@Eric-peng111  Reproducing is probably tricky... if you find a way to reproduce, please tell us and it will be assigned to you 🙂

**Comment by Eric-peng111** [comment]
@nicolas-raoul  
I am not sure if this is the issue where the button seems not working all the time no matter how many times I click on both
Besides, in my emulator, I cannot even use this 'review' function 
etc: Image cannot be shown no matter how long I wait
<img width="328" alt="image" src="https://user-images.githubusercontent.com/111075450/196629296-d404aa41-f0cc-4ac3-b072-8abb78f916b3.png">
